apps/mobile/src/app/wallet/(tabs)/_layout.tsx

```diff
@@ -3,15 +3,22 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import { ActionBarMethods } from '@/components/action-bar';
 import { ActionBarContainer, ActionBarContext } from '@/components/action-bar/container';
-import { Modal } from '@/components/bottom-sheet-modal';
 import { BlurredHeader } from '@/components/headers/containers/blurred-header';
 import { TabBar } from '@/components/tab-bar';
 import { APP_ROUTES } from '@/routes';
+import { useSettings } from '@/state/settings/settings.slice';
```

Comment:
We seem to import `import { BottomSheetModal } from '@gorhom/bottom-sheet';` in a lot of places and se it as a ref type:
```
 const recoverWalletModalRef = useRef<BottomSheetModal>(null);
```

Maybe I should move that into the UI package also and export the type from there? 

Then I can try and take away the dependancy `@gorhom/bottom-sheet` in `apps/mobile`

Reply:
I decided to go ahead and do this. Now we use `@gorhom/bottom-sheet` in the `UI` package and export what we need from there so we can remove the dependancy on it in the mobile app